lib/src/property.dart

```diff
@@ -529,9 +536,13 @@ class Rgba implements _StyleProperty, ColorBase {
   factory Rgba.fromColor(Color color) => color.rgba;
 
   factory Rgba.fromArgbValue(num value) {
-    return new Rgba(((value.toInt() & 0xff000000) >> 0x18), /* a */
-        ((value.toInt() & 0xff0000) >> 0x10), /* r */
-        ((value.toInt() & 0xff00) >> 8), /* g */
+    return new Rgba(
+        ((value.toInt() & 0xff000000) >> 0x18),
```

Comment:
I would change these comments so that they go above the expression they describe